### ISSUE TITLE
Retinanet R-50/R-101 on the COCO subsets has loss=nan

### ISSUE THREAD

**Issue by yuki111w** [open-mmlab/mmdetection#4864]
I was trying to train a retinanet model on COCO subset(person class only) with Resnet-50, Resnet-101 backbone with coco pretrained weights following the same setting as [cocoperson faster-rcnn](https://github.com/open-mmlab/mmdetection/tree/master/configs/faster_rcnn). However,all the losses became nan from the beginning of training. 
![log](https://user-images.githubusercontent.com/65905877/113095777-a9cb0800-9226-11eb-9d5c-81f2f80df72c.JPG)
I also tried training the one-stage detector GFL with Resnet-50 and  RetinaNet with Resnext-101 on the COCO-person data framework, getting 0.5660 and 0.5450  bbox_mAP respectively.

**Comment by yuki111w** [comment]
@hhaAndroid

**Comment by ZwwWayne** [comment]
Did you try to use pre-trained RetinaNet? This might because the randomly initialized retinanet is a little bit hard to learn.

**Comment by yuki111w** [comment]
@ZwwWayne I loaded the pre-trained retina weights of coco for all the models.

**Comment by ZwwWayne** [comment]
Then this means the model is a little bit unstable. We are not able to help here.

**Comment by Yuanmengyi918** [comment]
Hi, have you resolved this problem? I have the same problem as you.

**Comment by YSJ161017** [comment]
I have the same problem. When I don't use the pre-trained model, the loss is not non, but the acc will reach 16. When I use the pre-trained model, the loss is non，have you resolved this problem? @Yuanmengyi918 @yuki111w

**Comment by ZwwWayne** [comment]
https://mmdetection.readthedocs.io/en/latest/faq.html#training
